quinn/dataframe_helpers.py

```diff
@@ -1,10 +1,15 @@
-from typing import Any, Dict, List
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pyspark.sql import DataFrame, SparkSession
+from typing import Any
 
-from pyspark.sql import DataFrame, SparkSession
 from pyspark.sql.types import StructField, StructType
 
 
-def column_to_list(df: DataFrame, col_name: str) -> List[Any]:
+def column_to_list(df: DataFrame, col_name: str) -> list[Any]:
```

Comment:
Will it work with Python 3.7? Cause we are running tests fir spark 2 that requires python 3.7